shared/build.gradle.kts

```diff
@@ -59,6 +60,9 @@ kotlin {
         val commonTest by getting {
             dependencies {
                 implementation(kotlin("test"))
+                implementation(Dependency.COROUTINES_TEST)
+                implementation(Dependency.MOCKATIVE)
+
```

Comment:
extra new line?

Reply:
Fixed in https://github.com/suho/kmm-ic/pull/87/commits/fa346335edd4ca03014c9ddd5b53f6bd85f9e63a